packages/metasploit-framework/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_HOMEPAGE=https://metasploit.com/
+TERMUX_PKG_DESCRIPTION="Open source penetration testing framework"
+TERMUX_PKG_VERSION=4.16.4
+TERMUX_PKG_DEPENDS="autoconf, bison, clang, coreutils, curl, findutils, git, apr, apr-util, libffi-dev, libgmp-dev, libpcap-dev, postgresql-dev, readline-dev, libsqlite-dev, openssl-dev, libtool, libxml2-dev, libxslt-dev, ncurses-dev, pkg-config, postgresql-contrib, wget, make, ruby-dev, libgrpc-dev, termux-tools, ncurses-utils, ncurses, unzip, zip, tar"
+
+termux_step_make_install () {
+	echo "make install"
+	cd $TERMUX_PREFIX/share/
+
+	rm -rf $TERMUX_PREFIX/share/metasploit-framework
+	git clone https://github.com/rapid7/metasploit-framework --depth 1 
+	cd $TERMUX_PREFIX/share/metasploit-framework
+
+	# Remove data directory for now to avoid termux-elf-cleaner running on it
+	rm -rf data
+
+	# Skip rbnacl
+	sed '/rbnacl/d' -i Gemfile.lock
+	sed '/rbnacl/d' -i metasploit-framework.gemspec
+
+	# Skip grpc
+	sed '/grpc/d' -i Gemfile.lock
+	sed '/grpc/d' -i metasploit-framework.gemspec
+
+	#for MSF in $(ls msf*); do ln -s $MSF $TERMUX_PREFIX/bin/$MSF;done
+}
+
+termux_step_post_massage () {
+	echo "post massage"
+	cd $TERMUX_PREFIX/share/metasploit-framework
+
+	# put the data directory back
+	git checkout -- data/
+
+	gem install bundler -i $TERMUX_PREFIX/lib/ruby/gems/2.3.0
+	gem install nokogiri -- --use-system-libraries -i $TERMUX_PREFIX/lib/ruby/gems/2.3.0
```

Comment:
```
gem install nokogiri -i $TERMUX_PREFIX/lib/ruby/gems/2.3.0 -- --use-system-libraries
results in:
Successfully installed bundler-1.15.4
1 gem installed
Fetching: mini_portile2-2.2.0.gem (100%)
Successfully installed mini_portile2-2.2.0
Fetching: nokogiri-1.8.0.gem (100%)
Building native extensions with: '--use-system-libraries'
This could take a while...
ERROR:  Error installing nokogiri:
	ERROR: Failed to build gem native extension.

    current directory: /data/data/com.termux/files/usr/lib/ruby/gems/2.3.0/gems/nokogiri-1.8.0/ext/nokogiri
/usr/bin/ruby2.3 -r ./siteconf20170905-18028-1pmwjle.rb extconf.rb --use-system-libraries
mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h

extconf failed, exit code 1

Gem files will remain installed in /data/data/com.termux/files/usr/lib/ruby/gems/2.3.0/gems/nokogiri-1.8.0 for inspection.
Results logged to /data/data/com.termux/files/usr/lib/ruby/gems/2.3.0/extensions/x86_64-linux/2.3.0/nokogiri-1.8.0/gem_make.out
```